emva1288/camera/camera.py

```diff
@@ -150,6 +167,13 @@ def __init__(self,
         self._blackoffset = None
         self.blackoffset = blackoffset
 
+        self._transmition = {'green': transmition_green,
```

Comment:
Instead of having several variables here, just call `_u_bayer` here and store the pattern in `self.bayer` variable, or `None` if not bayer